netkan/netkan/webhooks/spacedock_inflate.py

```diff
@@ -0,0 +1,34 @@
+from flask import Blueprint, current_app, request, jsonify
+import re
+
+from ..metadata import Netkan
+from .common    import sqs_batch_entries
+
+
+spacedock_inflate = Blueprint('spacedock_inflate', __name__)
+
+
+# For after-upload hook on SpaceDock
+# Handles: https://netkan.ksp-ckan.space/sd/inflate?mod_id=1234&event_type=update
+@spacedock_inflate.route('/inflate', methods=['POST'])
+def inflate_hook():
+    # Make sure our NetKAN repo is up to date
+    current_app.config['netkan_repo'].remotes.origin.pull('master', strategy_option='ours')
+    # Get the relevant netkans
+    nks = find_netkans(request.form.get('mod_id'))
```

Comment:
The comment in line 12 is wrong, isn't it?
You specified POST as request type, and later on use `request.form.get()` which is also for POSTs, but in the code comment the data is passed as URL query parameter for GETs.

Reply:
Good point, that comment was meant to be a succinct example but it does falsely imply a `GET` operation. Will update it.